.gitignore

```diff
@@ -20,4 +20,5 @@
 # Go workspace file
 go.work
 
-build/
\ No newline at end of file
+build/
+main
```

Comment:
When running build it created an artefact in "main", I thought it'd be nice to never accidentally commit that